dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/consumer/TaskPriorityQueueConsumer.java

```diff
@@ -139,7 +140,13 @@ public void run() {
                 for (String dispatchFailedTask : failedDispatchTasks) {
                     taskPriorityQueue.put(dispatchFailedTask);
                 }
+                // If there are tasks in a cycle that cannot find the worker group,
+                // sleep for 1 second
+                if (!failedDispatchTasks.isEmpty()  && taskPriorityQueue.size() <= fetchTaskNum) {
```

Comment:
"taskPriorityQueue.size() <= failedDispatchTasks.size()" is more better.

Reply:
same.

Because the failed failedDispatchTasks queue can only have fetchTaskNum tasks at most

Reply:
it is not same.
the situation that do not need sleep. : failedNumber=1; queueSize=2; fetchTaskNumber=3

Reply:
> it is not same.
> the situation that do not need sleep. : failedNumber=1; queueSize=2; fetchTaskNumber=3

It has been dealt with, but now the main problem is to try many ways. Code coverage has not been able to